flags/flags.go

```diff
@@ -15,35 +15,37 @@ const (
 	minFlagKeyLen = 3 // Minimum flag key length helps reduce the number of false positives
 )
 
-func GetFlagKeys(opts options.Options, repoParams ld.RepoParams) []string {
-	projKey := opts.ProjKey
+func GetFlagKeys(opts options.Options, repoParams ld.RepoParams) map[string][]string {
+	flagKeys := make(map[string][]string)
+	for _, proj := range opts.Projects {
```

Comment:
Performing the loop in here because it already is coupled pretty tightly to implementation. Down the line we could look to move this outside of the function.